builtin/providers/aws/resource_aws_db_subnet_group.go

```diff
@@ -82,12 +83,17 @@ func resourceAwsDbSubnetGroupRead(d *schema.ResourceData, meta interface{}) erro
 	}
 
 	if len(describeResp.DBSubnetGroups) != 1 ||
```

Comment:
Got it – refactored in  db37b34 to fix this, take another look when you get a chance